doc/info/tips.3

```diff
@@ -440,19 +512,47 @@ File: TIPS,  Node: Create new bootable ITS, Previous: Copy user directories from
 
    d$nits
 
-6. Start ITS
+6. If you are using a PiDP-10, you must write-protect the disk headers.  To
+   do that, open the SIMH console as described above.  Type the following
+   commands:
+
+   ^E
+   set dpa noheaders
+   continue
+ 
+7. Start ITS
 
    g$
 
-7.  If ITS doesn't start, reenter DSKDMP:
+8.  If ITS doesn't start, reenter DSKDMP:
 
    $U
 
-8. And load and start the new ITS:
+9. And load and start the new ITS:
 
    nits
    $g
 
+10. On some systems, the nits file will not be on the correct disk pack.
+    It is critically important to get this right to avoid an unbootable
+    system.  For example, on PiDP-10 the file must reside on drive 0, pack 2.
```

Comment:
The system will not be unbootable.  You can mount packs in DSKDMP using the L$n$ command.